packages/passport/sdk/src/zkEvm/zkEvmProvider.ts

```diff
@@ -103,9 +112,13 @@ export class ZkEvmProvider implements Provider {
     });
 
     this.#multiRollupApiClients = multiRollupApiClients;
-    this.#eventEmitter = new TypedEventEmitter<ProviderEventMap>();
+    this.#providerEventEmitter = new TypedEventEmitter<ProviderEventMap>();
 
     passportEventEmitter.on(PassportEvents.LOGGED_OUT, this.#handleLogout);
+    passportEventEmitter.on(
+      PassportEvents.ACCOUNTS_REQUESTED,
+      trackSessionActivity,
+    );
```

Comment:
This should probably be executed in the constructor of `Passport.ts` where we initialise the `passportEventEmitter`